internal/util/util.go

```diff
@@ -390,3 +391,34 @@ func CallStack() string {
 
 	return string(stack)
 }
+
+// CheckSliceContains checks the slice for string.
+func CheckSliceContains(options []string, opt string) bool {
```

Comment:
`k8s.io/utils/strings/slices` is already used elsewhere, can you replace `CheckSliceContains()` by `slices.Contains()`?

Reply:
Wait, no! There is https://pkg.go.dev/slices in recent Golang versions :partying_face: , use that instead.

Reply:
done😃